test/crs/conversions.jl

```diff
@@ -350,6 +350,10 @@
       @inferred convert(LatLon, c2)
     end
 
+    # conversion LatLonAlt <> GeocentricLatLonAlt is tested in the next section
+    @testset "LatLonAlt <> GeocentricLatLonAlt" begin
+    end
```

Comment:
and these

Reply:
added test, I didn't initially put it cause they relay on the LatLon<>GeocentricLatLon conversion being correct and passing the altitude